packages/otelbin/src/components/monaco-editor/parseYaml.ts

```diff
@@ -240,8 +239,10 @@ export function isOtelColCRD(jsonData: IOtelColCRD) {
 }
 
 export function selectConfigType(config: string) {
-	const jsonData = JsYaml.load(config, { schema: FAILSAFE_SCHEMA }) as any;
-
+	let jsonData: any;
+	try {
+		jsonData = YAML.parse(config, { logLevel: "error", schema: "failsafe" }) as any;
+	} catch (e) {}
```

Comment:
**Question:** This silently catches and fully ignores the error. That seems dangerous to me. Why do we need to do this?

Reply:
I thought we were already displaying errors in the validation console; when editing the config, the console is overflowing with errors.

Reply:
Could you please document this in the code in both locations? Because this empty catch block very much looks like a bug otherwise :)

Reply:
The related comments added to the code